qemu/tests/trim_support_test.py

```diff
@@ -92,6 +100,11 @@ def _disk_size_smaller(ori_size):
     if status:
         test.error("Error when trim the volume, status=%s, output=%s" %
                    (status, output))
+    if event_id:
+        time.sleep(10)
+        session = vm.reboot(session)
+        if query_system_events(params['filter_options']):
+            test.fail("Disk corruption after trim for 4K")
```

Comment:
```suggestion
            test.fail("Disk corruption after trim for %s" % params.get("block_size"))
```
Then change the cfg:
```
             block_size = 4K
             physical_block_size_stg1 = 4096
             logical_block_size_stg1 = 4096
```

since if there is another block size requirement, such as 8k, 16k, and so on, you just add parameters in the cfg.